test/service/test_request_validation.py

```diff
@@ -87,11 +88,28 @@ def tearDown(self):
         if self.server_thread.is_alive():
             self.fail('Thread is still alive after joining')
 
-    def test_hello_world(self):
-        url = self.base_url
+    def test_health_es_reachable(self):
+        url = self.base_url + "health"
         response = requests.get(url)
+        expected_response = {
+            'status': 'UP',
+            'elasticsearch': {'domain': AzulConfig().es_domain, 'status': 'UP'}
+        }
         self.assertEqual(response.status_code, 200, response.json())
```

Comment:
This is bad. Please see my comment in the other PR.

Use `response.raise_for_status()`